internal/pkg/integration/store.go

```diff
@@ -137,20 +131,6 @@ func (m *MongodbConnectorStore) Uninstall(ctx context.Context, name string) erro
 	})
 }
 
-func (m *MongodbConnectorStore) FindAll(ctx context.Context) ([]payments.ConnectorBaseInfo, error) {
```

Comment:
what's the specific reason to remove this from the store / moving it to the api layer?

Reply:
Same question ^^